amxmodx/srvcmd.cpp

```diff
@@ -63,6 +63,44 @@ void amx_command()
 
 		print_srvconsole("%d plugins, %d running\n", plugins, running);
 	}
+	else if (!strcmp(cmd, "plugin"))
+	{
+		if (CMD_ARGC() < 3)
+		{
+			print_srvconsole("Usage: amxx plugin [ id ]\nFor a list of plugins, use the \"amxx plugins\" command\n");
+		}
+		else
+		{
+			auto id = atoi(CMD_ARGV(2));
```

Comment:
This can invoke UB. I think it's better to report to user if input is invalid

https://stackoverflow.com/questions/8871711/atoi-how-to-identify-the-difference-between-zero-and-error